desktop/core/src/desktop/models.py

```diff
@@ -1481,13 +1479,11 @@ def update_permission(self, user, name='read', users=None, groups=None, is_link_
 
     perm, created = Document2Permission.objects.get_or_create(doc=self, perms=name)
 
-    perm.users = []
```

Comment:
If users are Non shouldn't we clear() the list?

Reply:
If there are any good documents other than this https://docs.djangoproject.com/en/3.1/ref/models/relations/ please let me know so that I can understand this scenario properly.

Reply:
This doc is good, here in the comment above it seemed more like a logical flaw as we would skip the reset